tests/test_client.py

```diff
@@ -700,15 +699,16 @@ def _create_orders(self, num_records, start_date):
                     'recipient': {
                         'display_name': 'display name 42',
                     }
-                 },
-                 'state': 'PROPOSED'
+                },
+                'state': 'PROPOSED'
             }]
             created_orders.append(self.post_order(body, location_id).body.get('order'))
 
         return created_orders
 
     def create_shift(self, start_date, end_date, num_records):
-        employee_id = [employee['id'] for employee in self.get_all('employees', start_date)][0]
+        employee_id = self.get_first_found('employees', None)['id']
```

Comment:
I think this abstracts away the create. I think a better pattern is to have the logic in the method, but I know that isn't very DRY. Maybe the method name could be changed so that we know it creates a record if it can't find one.

Reply:
I'll rename it get_or_create_first_found in a subsequent PR